.github/workflows/prevalence.yml

```diff
@@ -1,47 +1,84 @@
 name: Prevalence
 
 on:
-  # schedule:
-  #   # 9:00am Pacific = 16:00 UTC
-  #   - cron:  '0 16 * * *'
+  schedule:
+    # 7:00am Pacific = 14:00 UTC
+    - cron:  '0 14 * * *'
   workflow_dispatch:
 
 jobs:
   prevalence:
-    # TODO(blanchardjeremy): This whole file is a work in progress.
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
+    - name: Checkout git repo (with full history)
+      uses: actions/checkout@v2
+      with:
+          fetch-depth: 0
 
-    - uses: actions/setup-python@v2
+    - name: Set up python
+      uses: actions/setup-python@v2
 
-    - uses: syphar/restore-virtualenv@v1
+    - name: Set up virtualenv
+      uses: syphar/restore-virtualenv@v1
       id: cache-virtualenv
       with:
         requirement_files: requirements-manual.txt
 
-    - uses: syphar/restore-pip-download-cache@v1
+    - name: Set up pip cache
+      uses: syphar/restore-pip-download-cache@v1
       if: steps.cache-virtualenv.outputs.cache-hit != 'true'
 
-      # the package installation will only be executed when the
-      # requirements-files have changed.
-    - run: pip install -r requirements.txt
+    - name: Install python dependencies
+      run: pip install -r requirements-manual.txt
       if: steps.cache-virtualenv.outputs.cache-hit != 'true'
 
-    - name: Run update preavlence
+    - name: Run prevalence update
+      env:
+        CAN_API_KEY: ${{ secrets.COVID_ACT_NOW_API_KEY }}
       run: |
-        CAN_API_KEY=COVID_ACT_NOW_API_KEY python3 update_prevalence.py
+        CAN_API_KEY="$CAN_API_KEY" python3 update_prevalence.py
 
-    # TODO: Delete branch we're going to use in case it exists
 
-    # Checkout a separate branch
-    
-    # Run update_prevalence.py including the CAN API KEY
-    
-    # Run linter (may need to set up all of yarn first?)
+    # We do this after the update_prevalence script in case we are running it from a branch (during testing)
+    - name: Create a branch
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        git stash
```

Comment:
Having a hard time following this step. Why are you pushing from local branch main to upstream branch auto-update-prevalence?

Reply:
This is a workaround to help it function when working on a branch (to update . We still wanna start the auto-prevalence-update branch based on main. Then we push to auto-prevalence-update local to remote.